contrib/scalar/t/t9099-scalar.sh

```diff
@@ -154,4 +154,13 @@ test_expect_success '`scalar clone` with GVFS-enabled server' '
 	)
 '
 
+test_expect_success '`scalar delete` without enlistment shows a usage' '
+	test_expect_code 129 scalar delete
+'
+
+test_expect_success '`scalar delete` with enlistment' '
+	scalar delete cloned &&
+	test ! -d cloned
+'
+
```

Comment:
Since tick marks mean something special in scripts, I have not seen them be used this way in the tests. The single quotes prevent them from being interpreted in these test names, but it is still a style thing.\

Also, use `test_path_is_missing` for cross-compatible checking of the missing directory.

```suggestion
test_expect_success 'scalar delete without enlistment shows a usage' '
	test_expect_code 129 scalar delete
'

test_expect_success 'scalar delete with enlistment' '
	scalar delete cloned &&
	test_path_is_missing cloned
'

```